springfox-spring-web/build.gradle

```diff
@@ -11,6 +11,7 @@ ext {
 
 dependencies {
   compile "org.reflections:reflections:$reflections"
+  compile files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
Is it ok if I call the project springfox-javadoc and use the springfox package names as it currently is doing? I would prefer to give this back to you as I won't be able to publish it to Maven Central using your groupId.